presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -32,6 +32,15 @@ The ``AT TIME ZONE`` operator sets the time zone of a timestamp::
 
     SELECT timestamp '2012-10-31 01:00 UTC' AT TIME ZONE 'America/Los_Angeles';
     2012-10-30 18:00:00.000 America/Los_Angeles
+    
+The ``AT TIME ZONE`` operator only supports a constant as a parameter. When using a field for timezone conversion, the at_timezone() function must be used. Here, the value of ``table.timezone_name`` is 'America/Los Angeles'::
+
+    SELECT timestamp '2012-10-31 01:00 UTC';
+    2012-10-31 01:00:00.000 UTC
+    
+    SELECT at_timezone(timestamp '2012-10-31 01:00 UTC', table.timezone_name);
+    2012-10-30 18:00:00.000 America/Los_Angeles
+    
```

Comment:
`at_timezone` function is still marked as hidden in `prestodb`. See https://github.com/prestodb/presto/blame/master/presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java#L269

You might want to backport this PR : https://github.com/prestosql/presto/pull/1612 which includes both the documentation change and unhiding part.

cc @rongrong who can help give us direction regarding this.

Reply:
Ya, let's back port this.